packages/storage-compat/test/unit/service.test.ts

```diff
@@ -0,0 +1,233 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import '../setup';
+import { expect } from 'chai';
+import { stub } from 'sinon';
+import * as modularStorage from '@firebase/storage';
+import { makeTestCompatStorage, fakeApp, fakeStorage } from '../utils';
+import {
+  FirebaseStorage,
+  getStorage,
+  FirebaseStorageError
+} from '@firebase/storage';
+import firebase from '@firebase/app-compat';
+import { StorageServiceCompat } from '../../src/service';
+import { FirebaseApp } from '@firebase/app-types';
+
+const DEFAULT_HOST = 'firebasestorage.googleapis.com';
+
+describe('Firebase Storage > Service', () => {
+  describe('useEmulator(host, port)', () => {
+    it('calls connectStorageEmulator() correctly', () => {
+      const connectStorageEmulatorStub = stub(
+        modularStorage,
+        'connectStorageEmulator'
+      ).callsFake(() => {});
+      const service = makeTestCompatStorage(fakeApp, fakeStorage);
+      service.useEmulator('test.host.org', 1234);
+
+      expect(connectStorageEmulatorStub).to.have.been.calledWithExactly(
+        fakeStorage,
+        'test.host.org',
+        1234
```

Comment:
```suggestion
        1234,
        {}
```
I ran into a test failure here while messing with this branch and I think the test needs this to pass because there's an `options` param with a default arg of `{}`.